test/unit/feature-test.js

```diff
@@ -1,14 +1,16 @@
-describe('A feature test', function() {
-  beforeEach(function() {
+import MyLibrary from '../../src/index';
+
+describe('A feature test', () => {
+  beforeEach(() => {
     spy(MyLibrary, 'mainFn');
     MyLibrary.mainFn();
   });
 
-  it('should have been run once', function() {
+  it('should have been run once', () => {
     expect(MyLibrary.mainFn).to.have.been.calledOnce;
   });
 
-  it('should have always returned hello', function() {
+  it('should have always returned hello', () => {
```

Comment:
Mocha inherits scope from the level above it. Remember when I "this-ified" all the Marionette tests? Using arrow functions all the way up would cause them to break.